astropath/bayesian.py

```diff
@@ -2,48 +2,56 @@
 
 import numpy as np
 
-import copy
-
 from astropy import units
 from astropy.coordinates import SkyCoord
 
+from astropath import chance
+
 from IPython import embed
 
 
-def raw_prior_Oi(Pchance, Sigma_m, method, half_light=None):
+def raw_prior_Oi(method, mag, half_light=None, Pchance=None):
     """
     Raw prior for a given set of Pchance values and/or n(m)
 
     Args:
-        Pchance (float or np.ndarray):
-            Chance probability
-        Sigma_m (float or np.ndarray):
-            Number density of sources on the sky brighter than m
-            number per sq arcsec
         method (str):
             inverse :: Assign inverse to Sigma_m
             inverse1 :: Assign inverse to Sigma_m * half_light
             inverse2 :: Assign inverse to Sigma_m * half_light**2
             pchance_inverse :: Assign inverse to P_chance
             identical :: All the same
+        mag (float or np.ndarray):
+            Magnitudes of the sources;  assumed r-band
         half_light (float or np.ndarray, optional):
             Angular size of the galaxy in arcsec
             Only required for several methods
+        Pchance (float or np.ndarray, optional):
+            Chance probability
+            Required for linear and orig_inverse methods
```

Comment:
linear is gone now. So you can remove it from this statement. :)

Reply:
removed